### ISSUE TITLE
No naviagtion after authenticating / followup after #3788 issue

### ISSUE THREAD

**Issue by LaVanguard** [FlutterFlow/flutterflow-issues#3830]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I have a button which autheticates and navigates the user to a selected page.
It logs the user via google, but doesn't navigate him.

### Expected Behavior

User is navigated after authetication like the flow says.

### Steps to Reproduce

Go to Admin Login
Click Sign In
Select google account which user has userLevel Enum as Admin.
Get signed in but not navigated.

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4sk876y4x1oeFF+Kr+acBr/SoXJm4+ap0SscsZa0kdB4joPLJ+d8jRYldBYM/gTENhHGKlkTsG0MrMi9ntF/coKQyrbIBE+c5QWj3yd3q8Rr6PDLWofXFSM9lgC1SZ4LaznxZCKtdZWCEk3GCIfOiyaA7DN5+/Zwh9f6PHaOI=

### Visual documentation

![image](https://github.com/user-attachments/assets/4d7e0460-5387-48a0-b8d6-b003add5c627)

### Environment

```markdown
- FlutterFlow version: FlutterFlow v4.1.89+ released September 12, 2024
- Platform: PC
- Browser name and version: Wersja 128.0.6613.138 (Oficjalna wersja) (64-bitowa)
- Operating system and version affected: Windows 11 23H2
```

### Additional Information

Previous ticket was closed before it was fixed. Please help!

**Comment by Alezanello** [comment]
Hello!

I’m closing this issue as it’s a duplicate of the following: [Issue #3788](https://github.com/FlutterFlow/flutterflow-issues/issues/3788).

Let’s continue the discussion and work on resolving it in the original post. Thanks for your understanding!
